### ISSUE TITLE
Built-in fourmolu does not respect yaml settings, external fourmolu does not launch

### ISSUE THREAD

**Issue by tablitza** [haskell/haskell-language-server#3968]
## Your environment 

Which OS do you use?
Arch
Which version of GHC do you use and how did you install it?
GHC 9.4.8, HLS 2.5.0.0 via ghcup

Which LSP client (editor/plugin) do you use?
emacs+lsp-mode
Have you configured HLS in any way (especially: a `hie.yaml` file)? No.

## What's wrong?

The built-in fourmolu plugin does not format code according to settings in yaml file. For instance, indentation parameter value has no impact at all. The yaml file does get loaded per the hls log file, and other parameters such as indent-wheres do in fact have an effect. When I switch to external plugin, I get the following error log:
`Error | fourmolu: Internal Error: fourmolu: createProcess: posix_spawnp: does not exist (No such file or directory)`
I installed fourmolu with cabal, it resides in `~/.cabal/bin`  which is on my PATH and launching it manually from terminal works.

**Comment by tablitza** [comment]
Indeed, it appears that the bundled fourmolu responds to the value in emacs standard-indent variable instead of the yaml file. fwiw, it would be convenient if there was an option to toggle this behavior as different projects may have their their style guide preference. I'm still curious why external fourmolu doesn't work.

**Comment by mngrm3a** [comment]
The `reexports` setting is ignored as well but `fixities` is not.
